sys/linux/futex.txt

```diff
@@ -17,4 +18,13 @@ robust_list {
 	next	ptr[in, robust_list, opt]
 }
 
+futex_wait_block {
+	val		int64
+	uaddr		ptr[in, int64]
+	bitset		const[0xffffffff, int32]
```

Comment:
Isn't bitset here, it's flags.

Reply:
and it isn't a const value, it can be `FUTEX_PRIVATE_FLAG` and it always need to have a `FUTEX_32` flag.

Reply:
Updated